scripts/ros-generate-cache.sh

```diff
@@ -0,0 +1,132 @@
+# /bin/sh (deliberately no !#)
+#
+# Usage: cd meta-ros
+#        sh scripts/ros-generate-cache.sh ROS_DISTRO YYYYMMDD PATH-TO-LOCAL-ROSDISTRO ROSDISTRO-COMMIT
+#            or
+#        sh scripts/ros-generate-cache.sh --version
+#
+#        Generate cache.yaml file for the YYYYMMDD release of ROS_DISTRO from the specified commit of a local
+#        ros/rosdistro.git . If running prior to first release of ROS_DISTRO, specify "none" for YYYYMMDD. NB. The release
+#        date might not match the commit timestamp of ROSDISTRO-COMMIT; however, the commit timestamp is used for the
+#        DATETIME when forming the name of the created branch.
```

Comment:
> when forming the name of the created branch

This script needs to be the one that creates the branch instead of superflore, because both commits will have to part of the PR. IMHO, the best way to have this happen would be to add an option to superflore that disables its creation of a branch. Can you make these changes?